tasks/gulp/copy-to-destination.js

```diff
@@ -25,5 +30,24 @@ gulp.task('copy-files', () => {
       autoprefixer
     ], {syntax: require('postcss-scss')}))
     .pipe(scssFiles.restore)
+    .pipe(yamlFiles)
+    .pipe(map(function (file, done) {
+      let componentName = path.dirname(file.path).split(path.sep).slice(-1).toString()
+      let componentPath = path.join(configPaths.components, componentName, `${componentName}.yaml`)
+
+      try {
+        let json = yaml.safeLoad(fs.readFileSync(componentPath), {encoding: 'utf8', json: true})
+        let paramsJson = {'params': json.params} // We only want the 'params' data from component yaml
```

Comment:
should we stop the task if the yaml file doesn't contain the `params` data

Reply:
This is in a `try` block so it will throw up an error if that happens and the tests will also fail. That's probably okay? This is in the middle of the `copy-files` task so if we exit here, there might already be files copied over to `package` and others deleted.